### ISSUE TITLE
Sqlite not creating database file

### ISSUE THREAD

**Issue by pwwang** [pwwang/pymedoo#7]
Hi,

I found sqlite engine stop working after the latest update. This code snippet will not create the db file. If I run it with earlier version to create the db file, and run it again with 0.0.6, `print(tables)` will print `[]`.

```
from medoo import Medoo

db = Medoo(dbtype="sqlite", database="z:\\test.db")
tables = db.query("SELECT name FROM sqlite_master WHERE type='table';")
tables = [x['name'] for x in tables.all()]
print(tables)
if 'books' not in tables:
    db.query('''create table books
        (id integer primary key,
        title varchar(25) not null default '',
        author varchar(6) not null default ''
        );''')
```

_Originally posted by @qijianjun in https://github.com/pwwang/pymedoo/issues/6#issuecomment-644268640_

**Comment by pwwang** [comment]
See: https://github.com/pwwang/pymedoo/commit/cdc77906f32e7c6dfc135043cdc5a10a7bebef37#r39922537
Will fix soon.

**Comment by pwwang** [comment]
FIx at [v0.0.7](https://github.com/pwwang/pymedoo/tree/0.0.7)
